public-ui/src/app/views/survey-view/survey-view.component.ts

```diff
@@ -82,7 +92,12 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
       this.searchText.valueChanges
         .debounceTime(1500)
         .distinctUntilChanged()
-        .subscribe((query) => { this.filterResults(); }));
+        .subscribe((query) => {
+          this.router.navigate(
```

Comment:
this one navigate should just be pulled to the top, may be. The search term was not showing up in survey url if this is just here. Pushing that change.

Reply:
thank you!